### PR TITLE
Simplfying the build using container builder privileged mode

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -5,13 +5,12 @@ TAG=$(date +%Y-%m-%d)
 REPO=$1
 VERSION=$2
 GCLOUD_CMD="gcloud"
-if [ -n $3 ]
+if [ "$3" != "" ]
```
Comment (nkubala) [inline build.sh]: a little nitpicky, but I believe this is the same as `[ -z "$3" ]`, which seems a little cleaner to me.

File: cloudbuild.yaml [hunk 2]
```diff
@@ -4,17 +4,11 @@ steps:
...
-                - '--tag=docker-mkdebootstrap'
-                - 'docker-mkdebootstrap'
-        # This is an intermediate step in order to run docker in privileged mode.
-        # It just pulls the mkdebootstrap image built just above and runs
-        # debootstrap on it.
-        - name: docker-mkdebootstrap
+        - name: %REPO%/mkdebootstrap:%VERSION%
           args:
-                - '%REPO%'
+                - '-d/workspace'
```
Comment (dlorenc) [inline cloudbuild.yaml]: This looks pretty strange, just double checking that there isn't supposed to be a space between the -d and /workspace

File: third_party/docker/README.md [hunk 3]
```diff
@@ -7,9 +7,3 @@ mkimage/debootstrap
 URL: https://github.com/docker/docker/tree/master/contrib/mkimage
 Date: 09/07/16
 Commit: e482d8472b7212f8e9fe13c33f12e8206393dcb5
-
-docker
```
Comment (dlorenc) [inline third_party/docker/README.md]: Was this deleted in the third_party dir?

File: build.sh (hunk 1 above)
Comment (sharifelgamal) [inline build.sh]: Fixed, I'm going to use -n properly.

File: cloudbuild.yaml (hunk 2 above)
Comment (sharifelgamal) [inline cloudbuild.yaml]: adding a space adds a space to the actual path it passes to argo which causes it to fail

File: third_party/docker/README.md (hunk 3 above)
Comment (sharifelgamal) [inline third_party/docker/README.md]: Yep, there's no need for a docker binary anymore.

**Pull request merged**